src/jsifier.js

```diff
@@ -423,6 +430,9 @@ function ${name}(${args}) {
       if (sig && (RELOCATABLE || ASYNCIFY == 2)) {
         contentText += `\n${mangled}.sig = '${sig}';`;
       }
+      if (isAsyncFunction) {
```

Comment:
Should we only output these `.isAsync` properties when `ASYNCIFY` is enabled?

Can `isAsyncFunction` ever be true when `ASYNCIFY` is not set?  i.e. what does it mean to have `AsyncFunction` or `__async` attribute when `ASYNCIFY` is not enabled?   Should we assert here maybe?

Reply:
I changed it to only output when ASYNCIFY is enabled.

> Can isAsyncFunction ever be true when ASYNCIFY is not set? i.e. what does it mean to have AsyncFunction or __async attribute when ASYNCIFY is not enabled? Should we assert here maybe?

Currently it won't ever be true, but yeah it should probably error if you're not using asyncify with async stuff. However, there may be some users that use `async fn()` in their libraries without asyncify.  I'm thinking I'll just add an assert for `$symbol__async`.